psi4/driver/driver_nbody.py

```diff
@@ -25,830 +25,1290 @@
 #
 # @END LICENSE
 #
-
+"""Plan, run, and assemble QC tasks to obtain many-body expansion and basis-set superposition error treatments.
+
+=============
+ManyBody Flow
+=============
+Bullet points are major actions
+Lines of dashes denote function calls
+e/d/dd=dg/g/h := energy, dipole, dipole derivative = dipole gradient, gradient, Hessian
+
+---------------------------
+ManyBodyComputer.__init__()
+---------------------------
+* not an explicit function but pydantic handles some defaults and validation
+* fields molecule, nfragments, bsse_type, return_total_data, and initial max_nbody set
+* BaseComputer.__init__()
+
+task_planner()
+--------------
+* computer gets modified from task_planner outside this file!
+* user levels kwarg processed into modelchem treatment levels and the n-body levels they cover. fields nbodies_per_mc_level set and max_nbody reset
+* for each modelchem treatment level, call build_tasks() below via one of four routes, depending on simple MB or layered MB(FD), MB(CBS), or MB(FD(CBS))
+
+    ------------------------------
+    ManyBodyComputer.build_tasks()
+    ------------------------------
+    * if supersystem requested, request (frag, bas) indices for full nbody range of nocp treatment from build_nbody_compute_list()
```

Comment:
I had to look up what "supersystem" and "nocp" meant. Also, what are frag and bas indices? I'm guessing that you index the fragments, but what is this about bases?

Reply:
added definitions like above